### PR TITLE
xToken snippet

### PR DESCRIPTION
**Pull request by jboetticher**
### Description

Added a code "snippet" to the xToken page so that people know how to use xTokens with libraries. I say with quotation marks since the snippets are scripts that are up to 80 lines long, which may warrant some shuffling of code

I did not add a link from any library pages to the xTokens page since I felt it would be too off-topic.

### Checklist

- [X] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [X] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [X] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [X] If images have been added, I have run the `compress-images.py` script to compress the images.
- [X] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: .snippets/code/xtokens/ethersjs.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+```javascript
+import abi from './xtokensABI.js'; // Import the xTokens ABI
```
Comment (eshaben) [inline .snippets/code/xtokens/ethersjs.md:2]: maybe we should have a snippet for the ABI too?

**Review by eshaben — COMMENTED** [review]
Looks good!! Not sure what you think about adding a snippet for the x-tokens ABI?

File: .snippets/code/xtokens/ethersjs.md (hunk 1 above)
Comment (jboetticher) [reply .snippets/code/xtokens/ethersjs.md:2]: I've added the ABI as a link

Comment (eshaben) [reply .snippets/code/xtokens/ethersjs.md:2]: perfect thanks!

**Review by eshaben — APPROVED** [review]
LGTM! Thanks!! 🚀

**Pull request merged**